activerecord/lib/active_record/fixture_set/render_context.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class ActiveRecord::FixtureSet::RenderContext # :nodoc:
```

Comment:
I have to use a compact style class definition here otherwise the render context changes and fails `ActiveRecord::FixtureSet::FileTest#test_render_context_lookup_scope` 🤔. How else can I fix this?

Reply:
Hrm I'm not sure how to fix this either. I think for now this is fine.